doc/overview/project_capabilities.md

```diff
@@ -12,6 +12,15 @@ known to declare the capability.
 | VisualC | Project may contain or compile C++ source files.
 | CSharp | Project may contain or compile C# source files.
 | VB | Project may contain or compile VB source files.
+| JavaScript | Project contains and handles JavaScript source files.
+| TypeScript | Indicates that the TypeScript build targets are imported into the project and the project is capable of compiling TypeScript source files into JavaScript.
+| JavaScriptProject | Indicates that the project supports JavaScript development. This capability also means that the project can manage JavaScript related project items and resolving JavaScript library references.
```

Comment:
On a second thought, I think the support for javascript library references is not accurate. What implemented in the project system is merely generic SDK references. And the one used often with JavaScript project is the WinJS library(which is a library in JavaScript). It's not quite a special type of "javascript library references".